admin/securedrop_admin/__init__.py

```diff
@@ -637,6 +637,7 @@ def load(self, validate=True):
         try:
             with io.open(self.args.site_config) as site_config_file:
                 c = yaml.safe_load(site_config_file)
+                self._config_in_progress = c
                 return self.clean_config(c) if validate else c
         except IOError:
             sdlog.error("Config file missing, re-run with sdconfig")
```

Comment:
With the validation changes and `validate=true`, this exception is never thrown. A `KeyError: `securedrop_app_gpg_public_key'` is thrown instead, which is a little misleading.

Reply:
This is unrelated  to the fix at hand tho and also already present in 1.4.0.

Reply:
I've added a fix for this as well.